### PR TITLE
Add bun extension

### PR DESCRIPTION
**Pull request by lafkpages**
## Description

This extension is a convenient way to read Bun change-logs and stay up to date on new versions, compare with your local version, search for Bun documentation and guides, and initialise new projects using `bun init`.

GitHub OAuth is used for making GraphQL calls to GitHub's API to fetch Bun versions and releases.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/bun/package.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        "--version",
+        "rev",
+        "revision",
+        "upgrade",
+        "update"
+      ]
+    },
+    {
+      "name": "blog",
+      "title": "Blog and Changelogs",
```
Comment (pernielsentikaer) [inline extensions/bun/package.json:29]:
```suggestion
      "title": "View Blog and Changelogs",
      "subtitle": "Bun",
```

File: extensions/bun/package.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  "icon": "extension_icon.png",
+  "author": "lafkpages",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "version",
+      "title": "Version",
```
Comment (pernielsentikaer) [inline extensions/bun/package.json:15]:
```suggestion
      "title": "Show Version",
      "subtitle": "Bun",
```

File: extensions/bun/package.json [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    },
+    {
+      "name": "blog",
+      "title": "Blog and Changelogs",
+      "description": "Read blog posts from bun.sh.",
+      "mode": "view"
+    },
+    {
+      "name": "docs",
+      "title": "Documentation and Guides",
```
Comment (pernielsentikaer) [inline extensions/bun/package.json:35]:
```suggestion
      "title": "Search Documentation and Guides",
      "subtitle": "Bun",
```

File: extensions/bun/package.json [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+          "placeholder": "Search"
+        }
+      ],
+      "keywords": [
+        "algolia"
+      ]
+    },
+    {
+      "name": "init",
+      "title": "Initialize New Project",
```
Comment (pernielsentikaer) [inline extensions/bun/package.json:51]:
```suggestion
      "title": "Initialize New Project",
      "subtitle": "Bun",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Im getting this error when I logged into GitHub and tried the `Version` command

```
09:53:03.122 Error: Minified React error #301; visit https://reactjs.org/docs/error-decoder.html?invariant=301 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Hl (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/react-reconciler/react-reconciler.production.min.js:5:18770)
    at Yl (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/react-reconciler/react-reconciler.production.min.js:7:3134)
    at vo (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/react-reconciler/react-reconciler.production.min.js:7:41965)
    at fo (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/react-reconciler/react-reconciler.production.min.js:7:36991)
    at Os (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/react-reconciler/react-reconciler.production.min.js:7:36922)
    at qr (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/react-reconciler/react-reconciler.production.min.js:7:36779)
    at vu (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/react-reconciler/react-reconciler.production.min.js:7:33275)
    at uo (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/react-reconciler/react-reconciler.production.min.js:7:32231)
    at Lu (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle<…>
09:53:03.206 
Error: Minified React error #301; visit https://reactjs.org/docs/error-decoder.html?invariant=301 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

Command:version.tsx:21:23

---
18: function Command() {
19:   const client = getGithubApi(getAccessToken());
20: 
21:   const localVersion = useLocalBunVersion();
22:   const latestVersion = useLatestBunVersion(client);
23: 
24:   useEffect(() => {
---

    at withAccessToken(Command)
_r:index.js:8:2593
    at ray-navigation-stack
```

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by Jarred-Sumner** [comment]
cool!

**Pull request closed without merging**